key-uglifier/src/uglifier.rs

```diff
@@ -0,0 +1,107 @@
+use colored::Colorize;
+use hex::ToHex;
+
+fn param_set_default_encryption_key(key_length: u8) -> Vec<u8>
+{
+    let default_key = vec![0x62u8; key_length as usize];
+    return default_key
+}
+
+fn read_hex_nibble(c: char) -> (bool, String)
+{
+    if (c >='0') && (c <= '9'){
+        return (true, c.to_string());
+    }
+    else if (c >='A') && (c <= 'F')
+    {
+        return (true, c.to_string());
+    }
+    else if (c >='a') && (c <= 'f')
+    {
+        return (true, c.to_string());
+    }
```

Comment:
aren't there a simpler way how to check if byte is hex?

smth like https://doc.rust-lang.org/std/primitive.u32.html#method.from_str_radix

or 
https://doc.rust-lang.org/std/primitive.char.html#method.is_digit

Reply:
I tried to save the original logic and flow in those places where I could do it.

Reply:
original to what?

Reply:
According to the source project with the wrong crypto implementation.

Reply:
> According to the source project with the wrong crypto implementation.

Could you post a reference/link to the source?

Reply:
> Could you post a reference/link to the source?

@Skarlett no, that's a closed repo.